core/ir/instr_api.h

```diff
@@ -2539,17 +2548,17 @@ instr_is_reg_spill_or_restore(void *drcontext, instr_t *instr, bool *tls DR_PARA
 /* we only care about these 11 flags, and mostly only about the first 6
  * we consider an undefined effect on a flag to be a write
  */
-#    define EFLAGS_READ_CF 0x00000001  /**< Reads CF (Carry Flag). */
-#    define EFLAGS_READ_PF 0x00000002  /**< Reads PF (Parity Flag). */
-#    define EFLAGS_READ_AF 0x00000004  /**< Reads AF (Auxiliary Carry Flag). */
-#    define EFLAGS_READ_ZF 0x00000008  /**< Reads ZF (Zero Flag). */
-#    define EFLAGS_READ_SF 0x00000010  /**< Reads SF (Sign Flag). */
-#    define EFLAGS_READ_TF 0x00000020  /**< Reads TF (Trap Flag). */
-#    define EFLAGS_READ_IF 0x00000040  /**< Reads IF (Interrupt Enable Flag). */
-#    define EFLAGS_READ_DF 0x00000080  /**< Reads DF (Direction Flag). */
-#    define EFLAGS_READ_OF 0x00000100  /**< Reads OF (Overflow Flag). */
-#    define EFLAGS_READ_NT 0x00000200  /**< Reads NT (Nested Task). */
-#    define EFLAGS_READ_RF 0x00000400  /**< Reads RF (Resume Flag). */
+#    define EFLAGS_READ_CF 0x00000001 /**< Reads CF (Carry Flag). */
```

Comment:
(I guess your clang-format is different -- but if the GA CI passed someone else would hit this anyway as it must be an older version vs newer.  Small enough to not need separation/tweaking; seems ok to leave in this PR.)

Reply:
Yes, this is a clang-format issue. I'm using version 9.0.0, not sure if that's old or new in tis context. I can always revert back if you'd like me to!

Reply:
We have documented that we're using version 14: https://dynamorio.org/page_code_style.html#autotoc_md122
Given that yours is older, yes please revert.  (If yours was newer we might consider updating.)

Reply:
Sorry, missed that. Fixed.